### PR TITLE
Update changelog for Rust 1.39 and 1.40

### PR DESCRIPTION
**Pull request by phansch**
[Rendered](https://github.com/phansch/rust-clippy/blob/update_changelog/CHANGELOG.md)

changelog: none

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,13 +8,99 @@ document.
 
 [3aea860...master](https://github.com/rust-lang/rust-clippy/compare/3aea860...master)
```
Comment (flip1995) [inline CHANGELOG.md]: This should also be updated:
```suggestion
 [69f99e7...master](https://github.com/rust-lang/rust-clippy/compare/69f99e7...master)
```

Not sure of this hash, see comment below

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,13 +8,99 @@ document.
...
-## Rust 1.39
+## Rust 1.41
 
 Current Beta
 
+## Rust 1.40
+
+Current stable, released 2019-12-19
+
+[4e7e71b...69f99e7](https://github.com/rust-lang/rust-clippy/compare/4e7e71b...c8e3cfb)
```
Comment (flip1995) [inline CHANGELOG.md]: second hash in link name and actual link differ.

**Comment by phansch** [comment]
@flip1995 good catch, thanks! should be fixed now.

**Comment by flip1995** [comment]
Thanks!

@bors r+

**Pull request merged**